### PR TITLE
(GH11) Implemented wyam documentation

### LINKED ISSUE #11: Start using wyam to document the addin

### PR DESCRIPTION
**Pull request by AdmiringWorm**
fixes #11 
@gep13 do this look about right for getting wyam up and running?

### CODE REVIEW

File: docs/input/docs/building/linux.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+Order: 2
+Title: Building on Linux
+Author: Kim Nordmo
+---
+
+## Requirements
+
+The following are need to build Cake.Transifex on Windows:
```
Comment (gep13) [inline docs/input/docs/building/linux.md]: @AdmiringWorm think you have a copy/pasta here...

Should this be Linux?

File: docs/input/docs/usage/index.cshtml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+Order: 2
+Description: How to obtain, configure, and execute Cake.Incubator.
```
Comment (gep13) [inline docs/input/docs/usage/index.cshtml]: @AdmiringWorm Should this be Transifex?

**Comment by gep13** [comment]
@AdmiringWorm have you ran this locally using `.\build.ps1 -target preview` to verify the navigation between pages?

File: docs/input/docs/building/linux.md (hunk 1 above)
Comment (AdmiringWorm) [inline docs/input/docs/building/linux.md]: yup, that's definitely a copy/pasta error.

File: docs/input/docs/usage/index.cshtml (hunk 2 above)
Comment (AdmiringWorm) [inline docs/input/docs/usage/index.cshtml]: Ops, well. At least you know why addin I used as a templates 😅

**Comment by AdmiringWorm** [comment]
@gep13 yeah, I did test it out with `.\build -target preview`, and as far as I could tell everything worked.
But I may perhaps have missed something, is there any pages you had in mind that perhaps won't work?

**Comment by AdmiringWorm** [comment]
One thing I found annoying though, is the javascript files located in the assets folder is never referenced, so they seem pretty much useless, right?

**Comment by AdmiringWorm** [comment]
@gep13 would you mind taking it for a test run?
Maybe you'll spot something that I'm unable to?

**Comment by gep13** [comment]
@AdmiringWorm baby duty just now, but should be on computer for a little bit later tonight.

**Comment by gep13** [comment]
>@AdmiringWorm said...
But I may perhaps have missed something, is there any pages you had in mind that perhaps won't work?

No, no, nothing wrong, I was just wondering if you had ran it locally, that's all 😄 Just ran the preview locally, and it looks good to me.  Just a couple grammatical things i noticed once ready the previewed version.

>@AdmiringWorm said...
One thing I found annoying though, is the javascript files located in the assets folder is never referenced, so they seem pretty much useless, right?

Ah, looks like I have missed something in the Cake.Incubator docs 😢 Probably the best docs site to have a look at is this one .  In there, you will see that the JavaScript files are included in the _Bottom.cshtml (https://github.com/cake-contrib/Cake.Prca.Website/blob/develop/input/_Bottom.cshtml).  This provides functionality for anchors of doc pages similar to this https://cake-contrib.github.io/Cake.Prca.Website/docs/overview/features#supported-core-functionality and also the ability to copy code snippets (https://cake-contrib.github.io/Cake.Prca.Website/docs/usage/basic-usage hover the code to get the copy button to appear)

File: docs/input/index.cshtml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Title: Cake.Transifex
+NoSidebar: true
+NoContainer: false
+NoGutter: false
+---
+
+<div class="container-fluid">
+	<h1>What is it?</h1>
+    <p>
+        Cake.Transifex is a addin for the Cake Build script adding support for working with the localization service Transifex.
```
Comment (gep13) [inline docs/input/index.cshtml]: Cake.Transifex is an addin...

File: docs/input/index.cshtml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        This addin requires that the transifex client is already installed and is available as `tx`.
+    </p>
+
+    <h2>Where to get it</h2>
+    <p>
+        Officially published versions are available on
+        <a href="https://www.nuget.org/packages/Cake.Transifex/" target="_blank">
+            NuGet
+        </a>.
+        While development versions is made available using the following nuget end point:
```
Comment (gep13) [inline docs/input/index.cshtml]: While development versions are made...

**Comment by AdmiringWorm** [comment]
@gep13 I'm honestly surprised there aren't more grammatical errors.
Anyways, I've updated the PR now, mind having a look over the changes?

**Comment by gep13** [comment]
@AdmiringWorm I think this is ready, great work! 👍 

Just to bring your attention to one thing...

For this section:

image

It might be worth thinking about using one of these callouts to bring more attention to it (http://www.digitaltapestry.net/posts/alerts-in-wyam).  This isn't a requirement, just a suggestion.

**Comment by AdmiringWorm** [comment]
@gep13 oh nice, I did not know that wyam supported that kind of markdown.
I'll change that at the same time as I merge this one in.

**Comment by AdmiringWorm** [comment]
Also need to look into why codecov doesn't report back that it have completed (the status checks), before I merge my PR.

**Pull request merged**